### PR TITLE
add extend_volume request to Fog::Volume::OpenStack

### PR DESCRIPTION
**Pull request by majewsky**
I would've liked to include tests, but the mock service is not up to it.
I cannot even create a volume and retrieve it again using the same ID.

### CODE REVIEW

**Comment by geemus** [comment]
@Ladas @mdarby could you guys take a peak? Thanks!

**Comment by dhague** [comment]
I'll have a look with @majewsky tomorrow to see about using VCR-based tests for this (we are both at SAP, albeit in different countries)

**Comment by geemus** [comment]
@dhague sounds great, just let me know if you need anything. Thanks!

**Comment by TerryHowe** [comment]
looks good, but more tests would always be nice.

**Comment by Ladas** [comment]
looks good, the VCR tests would be great, at least some form of tests should be present

**Comment by geemus** [comment]
Thanks for taking looks, just let me know once another pass has been taken and I'll review anew. Thanks!

**Comment by majewsky** [comment]
I rebased on master and added tests.

File: lib/fog/openstack/requests/volume/extend_volume.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  module Volume
+    class OpenStack
+      class Real
+        def extend_volume(volume_id, size)
+          body = { 'os-extend' => { 'new_size' => size } }
+          request(
+            :expects  => 202,
+            :method   => 'POST',
+            :path     => "volumes/#{volume_id}/action",
+            :body     => Fog::JSON.encode(body),
```
Comment (Ladas) [inline lib/fog/openstack/requests/volume/extend_volume.rb]: ruby 1.8.7 has problems with trailing commas

**Comment by majewsky** [comment]
I really understand now why you would want to drop 1.8.7 support. :)

**Comment by geemus** [comment]
Yeah, it can be a real pain. Thanks!

**Pull request merged**